### PR TITLE
Add social contents

### PR DESCRIPTION
**Pull request by Masako-KUNII**
I accidently deleted a few pages.
So I added those pages agein.

### CODE REVIEW

File: resources/views/social/contents/body.blade.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            <!--Body-->
+            <div class="modal-body d-flex justify-content-center align-items-center flex-column">
+
+                <div class="modal-body d-flex justify-content-center align-items-center flex-column">
+                    <span><i class="fa-solid fa-trash-can fa-3x icon-red"></i></span>
+                    <p class="pt-3 pr-2">Are you sure to want to delete this post?</p>
+                </div>
+
+                <div class="mt-3">
+                    <img src="https://images.pexels.com/photos/1435903/pexels-photo-1435903.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="#" class="delete-post-img img-fluid">
```
Comment (Ninit29) [inline resources/views/social/contents/body.blade.php:78]: Is this temporary?

File: resources/views/social/contents/body.blade.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<div class="container p-0 bg-white">
+    <a href="#">
+        <img src="https://images.pexels.com/photos/1435903/pexels-photo-1435903.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" alt="italian" class="w-100">
```
Comment (Ninit29) [inline resources/views/social/contents/body.blade.php:3]: Is this temporary? are you gonna upload your own image?

File: resources/views/social/contents/body.blade.php (hunk 1 above)
Comment (Masako-KUNII) [reply resources/views/social/contents/body.blade.php:78]: Here is the image of the post that the user wants to delete.
At the moment, image is placed temporarily for UI design.

File: resources/views/social/contents/body.blade.php (hunk 2 above)
Comment (Masako-KUNII) [reply resources/views/social/contents/body.blade.php:3]: This is temporary image for UI design.
User will post  own image.

**Pull request merged**